Amass/README.md

```diff
@@ -0,0 +1,30 @@
+## Amass Command
+
+```
+/bin/amass enum --passive -d
```

Comment:
keep `/bin/amass` and move `enum --passive -d` to the Arguments section

`enum --passive -d {{domain}}`